src/main/scala/tv/teads/tour/basics/FinalExam.scala

```diff
@@ -0,0 +1,13 @@
+package tv.teads.tour.basics
+
+object FinalExam {
+
+  // Given a minimum eligible price and a list of Bidders, compute the winner of the auction
+
+  case class Bidder(name: String, bid: BigDecimal)
+
+  def computeWinner(floorPrice: BigDecimal)(bidders: List[Bidder]): Option[Bidder] = {
```

Comment:
What's the purpose of using multiple parameter lists there ?

Reply:
I can add a question : given a floorPrice FP, create a function that takes the bidders and find an optional winner

Reply:
Reusing this function